integration/admin/admin_integration_test.go

```diff
@@ -92,9 +171,21 @@ func setupAdminServer() (trillian.TrillianAdminClient, func(), error) {
 	}
 	// lis is closed via returned func
 
+	// TODO(alanparra): Have a standard way to get a registry for tests. With a few changes
+	// we could leverage the utilities under testonly/integration.
+	db, err := mysql.OpenDB(*builtin.MySQLURIFlag)
```

Comment:
Yeah, it would be good not to use global flags, and not to use the default database.  Any chance you could fix this one before committing?